crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -628,6 +628,59 @@ pub struct UnableToDecryptInfo {
     /// `m.megolm.v1.aes-sha2` algorithm.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub session_id: Option<String>,
+
+    /// Reason code for the decryption failure
+    #[serde(default = "unknown_utd_reason")]
+    pub reason: UnableToDecryptReason,
+}
+
+fn unknown_utd_reason() -> UnableToDecryptReason {
+    UnableToDecryptReason::Unknown
+}
```

Comment:
For what it's worth, you can instead `derive(Default)` to `UnableToDecryptReason`, and then mark one variant as the default:

```rust
#[default]
#[doc(hidden)]
Unknown,
```